docs/skopeo-copy.1.md

```diff
@@ -98,7 +98,7 @@ To copy and sign an image:
 
 To encrypt an image:
 ```sh
-skopeo copy docker://docker.io/library/nginx:latest oci:local_nginx:latest
+skopeo copy docker://docker.io/library/nginx:1.17.8 oci:local_nginx:1.17.8
```

Comment:
Why this change? (It’s not obviously _undesirable_, but it seems quite unrelated and I can’t see why it is necessary.)

Reply:
I added an addition section, https://github.com/containers/skopeo/pull/814/files#diff-30c826204072774bd69d2ae15e60aef9R128, to show the partial image encryption in that file. 

The example image, `nginx:1.17.8` has exactly 3 layers and in order to showcase partial image encryption I encrypt the 2nd layer while keep the 1st and 3rd layer as is. While it didn't matter earlier because we were encrypting all the layers, this time I wanted to make sure that the image has exactly same 3 layers. We do not have control over how nginx image is built in future, maybe they could just shrink it to just one single layer. That's why I replaced `latest` with `1.17.8` tag. 

This will keep this doc unaffected by any future changes with respect to number of layers in the nginx image.

Reply:
Ah, I didn’t realize this is directly connected to the new example. The new example does benefit from fixing a version, sure.